src/jit/flowgraph.cpp

```diff
@@ -13490,13 +13490,15 @@ bool Compiler::fgOptimizeSwitchBranches(BasicBlock* block)
         GenTree* switchVal = switchTree->gtOp.gtOp1;
         noway_assert(genActualTypeIsIntOrI(switchVal->TypeGet()));
 
+#ifndef LEGACY_BACKEND
```

Comment:
Yes, it seems the various size / count / etc flags are never turned on and I don't think a single one would build until I started cleaning them up. I assume that LEGACY will go away at some point and this will become moot, but in the meantime one (admittedly small) benefit is that it's at least clear - wherever the operator is referenced - that it's only used by one version of the back end. Hope this makes some sense at least. Why there are different operators for these is the more fundamental question but that feels beyond the scope of this little change ;).